src/components/button/Button.type.ts

```diff
@@ -0,0 +1,8 @@
+export const BUTTON_VARIANTS = {
+    DIGIT: 'digit',
+    OPERATION: 'operation',
+    MODIFIER: 'modifier'
+  } as const
+
+  export type ButtonVariantType =
+  (typeof BUTTON_VARIANTS)[keyof typeof BUTTON_VARIANTS];
```

Comment:
여기 불필요한 들여쓰기가 있는 것 같아요.
+ eol에 개행 필요하다고 github이 표시해주네요.

```suggestion
export type ButtonVariantType =
(typeof BUTTON_VARIANTS)[keyof typeof BUTTON_VARIANTS];

```